homeassistant/components/light/tradfri.py

```diff
@@ -257,9 +257,9 @@ def async_turn_on(self, **kwargs):
             self.hass.async_add_job(self._api(
                 self._light_control.set_kelvin_color(kelvin)))
 
-        keys = {}
-        if ATTR_TRANSITION in kwargs:
-            keys['transition_time'] = int(kwargs[ATTR_TRANSITION])
+        keys = {
+            'transition_time': float(kwargs.get(ATTR_TRANSITION, 0.5)) * 10
```

Comment:
Is the underlying pytradfri library fine with 'transition_time' to be float?

Reply:
Good spot, it takes an int.